Doc/library/collections.rst

```diff
@@ -358,7 +358,7 @@ Common patterns for working with :class:`Counter` objects::
     list(c)                         # list unique elements
     set(c)                          # convert to a set
     dict(c)                         # convert to a regular dictionary
-    c.items()                       # convert to a list of (elem, cnt) pairs
+    c.items()                       # create a view of the dictionary’s items (elem, cnt) pairs
```

Comment:
Maybe this is more in line with the rest of the examples
```suggestion
    list(c.items())                 # convert to a list of (elem, cnt) pairs
```

Reply:
I disagree.  The doc for c.items() should say what c.items returns.